frontend/components/vault/WithdrawDaiFromVat.vue

```diff
@@ -0,0 +1,112 @@
+<template>
+    <BasePanel :current-state="currentStateAndTitle.name">
+        <template #title>{{ currentStateAndTitle.title }}</template>
```

Comment:
Why was this component copied over? I can see the identical `WithdrawDAIPanel` that should've been reused instead